src/server/page_server.cpp

```diff
@@ -184,6 +188,26 @@ page_server::connection_handler::handle_oldest_active_mvccid_request (tran_serve
   m_conn->respond (std::move (a_sp));
 }
 
+
+void
+page_server::connection_handler::receive_start_catch_up (tran_server_conn_t::sequenced_payload &&a_sp)
+{
+  auto payload = a_sp.pull_payload ();
+  cubpacking::unpacker unpacker { payload.c_str (), payload.size ()};
+
+  std::string host;
+  int32_t port;
+  LOG_LSA catchup_lsa;
+
+  unpacker.unpack_string (host);
+  unpacker.unpack_int (port);
+  cublog::lsa_utils::unpack (unpacker, catchup_lsa);
+
+  er_log_debug (ARG_FILE_LINE, "receive_start_catch_up: hostname: %s, port : %d, LSA (%lld|%d)\n", host.c_str (), port,
```

Comment:
Nit.
```suggestion
  er_log_debug (ARG_FILE_LINE, "receive_start_catch_up: hostname = %s, port = %d, LSA = (%lld|%d)\n", host.c_str (), port,
```
makes it easier to search in log